stopcovid/status/status.py

```diff
@@ -1,15 +1,27 @@
+import itertools
+from typing import List
+
 from stopcovid.dialog.types import DialogEventBatch
 from .drill_instances import DrillInstanceRepository
+from .initiation import trigger_first_drill
 from .users import UserRepository
+from ..dialog.dialog import UserValidated
 
 
-def handle_dialog_event_batch(batch: DialogEventBatch):
+def handle_dialog_event_batches(batches: List[DialogEventBatch]):
     user_repo = UserRepository()
 
-    # not worrying a lot about transactions here. All of these operations are idempotent
-    # and we're operating in a context in which we get only one event per phone number at a time.
+    # trigger initiation before updating status. The status updates could be slow because of
+    # aurora cold start time.
+    trigger_first_drill(phone_numbers_that_need_first_drill(batches))
+
+    for batch in batches:
+        user_id = user_repo.update_user(batch)
+
+        drill_instance_repo = DrillInstanceRepository()
```

Comment:
We could initialize this once outside of the loop, rather than at every iteration